storybook/src/stories/Form/TextAreaField.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+import {  text, select } from "@storybook/addon-knobs";
+import { TextAreaField } from 'scorer-ui-kit';
+
+const Container = styled.div`
+  margin: 20px;
+`;
+
+export default {
+  title: 'Form/molecules',
```

Comment:
The other Input stories in Storybook use the title `Form/Input` which I think is probably better than using molecules. I agree though that perhaps `Input` isn't the best subfolder for this. Perhaps we can combine the two stories in `Form/Input` with this one under a different name. Perhaps `Form/Fields` or something like that? Maybe even just putting them all under `Form/molecules` would be good for now. 

What do you think?

Reply:
Hi! I'm glad you mention it, I was not sure where to add it actually.
I saw that **TextField** on Inputs, but **Small Input** was an atoms.

Technically speaking all the items under Form are "Fields" so I wouldn't go for Form/Fields.

I have no problem in adding it to Form/Input if it's just to have less  folders.
Also let me know if you are interested that I move Small Input to Inputs in this PR.

Reply:
I'll add the sorting to a different issue in BugHerd and we and tackle that soon. Don't want to hold this merge up and fixing it later shouldn't cause any problems.